include/SDL3/SDL_video.h

```diff
@@ -920,6 +934,52 @@ extern DECLSPEC SDL_Window *SDLCALL SDL_GetWindowParent(SDL_Window *window);
 /**
  * Get the properties associated with a window.
  *
+ * The following properties are provided by SDL:
+ *
+ * On Android:
+ * "SDL.window.android.window" - the ANativeWindow associated with the window
+ * "SDL.window.android.surface" - the EGLSurface associated with the window
+ *
+ * On iOS:
+ * "SDL.window.uikit.window" - the (__unsafe_unretained) UIWindow associated with the window
+ * "SDL.window.uikit.metal_view_tag" - the NSInteger tag assocated with metal views on the window
+ *
+ * On KMS/DRM:
+ * "SDL.window.kmsdrm.dev_index" - the device index associated with the window (e.g. the X in /dev/dri/cardX)
+ * "SDL.window.kmsdrm.drm_fd" - the DRM FD associated with the window
+ * "SDL.window.kmsdrm.gbm_dev" - the GBM device associated with the window
+ *
+ * On macOS:
+ * "SDL.window.cocoa.window" - the (__unsafe_unretained) NSWindow associated with the window
+ * "SDL.window.cocoa.metal_view_tag" - the NSInteger tag assocated with metal views on the window
+ *
+ * On Vivante:
+ * "SDL.window.vivante.display" - the EGLNativeDisplayType associated with the window
+ * "SDL.window.vivante.window" - the EGLNativeWindowType associated with the window
+ * "SDL.window.vivante.surface" - the EGLSurface associated with the window
+ *
+ * On UWP:
+ * "SDL.window.winrt.window" - the IInspectable CoreWindow associated with the window
+ *
+ * On Windows:
+ * "SDL.window.win32.hwnd" - the HWND associated with the window
+ * "SDL.window.win32.hdc" - the HDC associated with the window
+ * "SDL.window.win32.instance" - the HINSTANCE associated with the window
+ *
+ * On Wayland:
+ * "SDL.window.wayland.display" - the wl_display associated with the window
+ * "SDL.window.wayland.surface" - the wl_surface associated with the window
+ * "SDL.window.wayland.egl_window" - the wl_egl_window associated with the window
+ * "SDL.window.wayland.xdg_surface" - the xdg_surface associated with the window
+ * "SDL.window.wayland.xdg_toplevel" - the xdg_toplevel role associated with the window
+ * "SDL.window.wayland.xdg_popup" - the xdg_popup role associated with the window
+ * "SDL.window.wayland.xdg_positioner" - the xdg_positioner associated with the window, in popup mode
+ *
```

Comment:
Should note here that the xdg_* types don't persist across show/hide calls and should be queried each time the window is shown.
```suggestion
 *
 * Note: The xdg_* window objects do not internally persist across window show/hide calls.
 *       They will be null if the window is hidden and must be queried each time it is shown.
 *
```